### PR TITLE
Update guidelines about creating a new repository

### PR DESCRIPTION
**Pull request by Ressetkk**
Updated new repository template and community guidelines that reflect new review and approval flow with Prow.

**Related issue(s)**

#642

### CODE REVIEW

File: docs/guidelines/repository-guidelines/01-new-repository-settings.md [hunk 1]
```diff
@@ -91,9 +50,6 @@ If you define any governance-related [Prow job](https://github.com/kyma-project/
...
-
-> **TIP:** You can copy labels from the existing repository to the new repository using the [GitHub CLI tool](https://github.com/jvandemo/copy-github-labels-cli).
+Labels are managed by Prow in the [`labels.yaml`](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml) configuration file. To add additional labels, simply raise a Pull Request to the test-infra repository with new label definition. Please follow the structure of this file to add a label.
 
-## Add the repository to teams
-Add the repository to both [developers](https://github.com/orgs/kyma-project/teams/developers/repositories) and [developers-triage](https://github.com/orgs/kyma-project/teams/developers-triage/repositories) teams.
+Additionally, you can [define repository exclusive labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories such as [`kyma`](https://github.com/kyma-project/kyma/labels).
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/01-new-repository-settings.md:55]:
```suggestion
Additionally, you can [define repository exclusive labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories in Kyma, such as the [`kyma`](https://github.com/kyma-project/kyma/labels) repository.
```

[the remark this answers is not in the document]
Comment (Ressetkk) [reply docs/guidelines/repository-guidelines/01-new-repository-settings.md:55]: I meant labels that are valid only for given repository. I believe `repository specific labels` or just `repository labels` would be enough to describe that.

File: templates/repository-template/OWNERS [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#   - githubUser2
+#   - githubUserN
+# approvers:
+#   - githubApprover1
+#   - githubApprover2
+#   - girhubApproverN
+# labels: # optional
+#   - label1
+#   - label2
+# Having the same set of reviewers and approvers is considered a bad practice and should be avoided!
```
[the remark this answers is not in the document]
Comment (Ressetkk) [reply templates/repository-template/OWNERS:17]: Reviewers look for general code quality, correctness, sane software engineering, style, etc.
Approvers look for holistic acceptance criteria, including dependencies with other features, forwards/backwards compatibility, API and flag definitions, etc.
When a person is in both reviewers and approvers groups, their Approving review is also considered as an approve and marks the PR as `approved`. That behaviour skips the requirement for a second review and immediately allows the PR to be merged.

File: docs/guidelines/repository-guidelines/01-new-repository-settings.md [hunk 3]
```diff
@@ -91,9 +52,6 @@ If you define any governance-related [Prow job](https://github.com/kyma-project/
...
-
-> **TIP:** You can copy labels from the existing repository to the new repository using the [GitHub CLI tool](https://github.com/jvandemo/copy-github-labels-cli).
+Labels are managed by Prow in the [`labels.yaml`](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml) configuration file. To add a new label, create a Pull Request to the [`test-infra`](https://github.com/kyma-project/test-infra) repository with a new label definition. When adding a new label, follow the structure of the `labels.yaml` file.
 
-## Add the repository to teams
-Add the repository to both [developers](https://github.com/orgs/kyma-project/teams/developers/repositories) and [developers-triage](https://github.com/orgs/kyma-project/teams/developers-triage/repositories) teams.
+Additionally, you can [define repository labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories such as [`kyma`](https://github.com/kyma-project/kyma/labels).
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/01-new-repository-settings.md:57]: As discussed: please update this paragraph and/or explain when we should create labels using PRs to the `test-infra` repo, and when using the method described in this article.

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 4]
```diff
@@ -3,26 +3,28 @@ title: Repository template
 label: internal
 ---
 
-The [`repository-template`](https://github.com/kyma-project/community/tree/main/templates/repository-template) folder offers a unified file, document, and folder structure. Use it for every new repository that you create in Kyma. It helps you to ensure that the project is consistent and standardized.
+The [`template-repository`](https://github.com/kyma-project/template-repository) repository offers a unified file, document, and folder structure. Use it for every new repository that you create in Kyma. It helps you to ensure that the project is consistent and standardized.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:6]:
```suggestion
The [`template-repository`](https://github.com/kyma-project/template-repository) offers a unified file, document, and folder structure. Use it for every new repository that you create in Kyma. It helps you to ensure that the project is consistent and standardized.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 5]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
+The `template-repository` contains all elements required for a skeleton repository. However, before you create a new repository from this template, read carefully the following paragraph to learn what the purpose of the specific files and documents is and which of them you need to adjust.
 
-The `repository-template` folder consists of:
+The `template-repository` consists of:
 
-* [`.github`](https://github.com/kyma-project/community/tree/main/templates/repository-template/.github) - This folder contains the pull request template, issue templates, and the Stale Bot that monitors inactive issues, marks them as `stale`, and closes them after the specified period of time.
+* [`.github`](https://github.com/kyma-project/template-repository/.github) - This folder contains the pull request template, issue templates, and the Stale Bot that monitors inactive issues, marks them as `stale`, and closes them after the specified period of time.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:14]:
```suggestion
* [`.github`](https://github.com/kyma-project/template-repository/tree/main/.github) - This folder contains the pull request template, issue templates, and the Stale Bot that monitors inactive issues, marks them as `stale`, and closes them after the specified period of time.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 6]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
 
-* [`docs`](https://github.com/kyma-project/community/tree/main/templates/repository-template/docs) - In this folder, put the repository-specific documentation only. Store any architectural decisions or documents applicable to all Kyma repositories in the `community` repository.
+* [`docs`](https://github.com/kyma-project/template-repository/docs) - In this folder, put the repository-specific documentation only. Store any architectural decisions or documents applicable to all Kyma repositories in the `community` repository.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:16]:
```suggestion
* [`docs`](https://github.com/kyma-project/template-repository/tree/main/docs) - In this folder, put the repository-specific documentation only. Store any architectural decisions or documents applicable to all Kyma repositories in the `community` repository.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 7]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
 
-* [CODE_OF_CONDUCT.md](https://github.com/kyma-project/community/blob/main/templates/repository-template/CODE_OF_CONDUCT.md) - This document is a ready-to-use template which provides a link to the general `CODE_OF_CONDUCT.md` document from the `community` repository. Copy the template into your own repository.
+* [CODE_OF_CONDUCT.md](https://github.com/template-repository/CODE_OF_CONDUCT.md) - This document is a ready-to-use template which provides a link to the general `CODE_OF_CONDUCT.md` document from the `community` repository.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:18]:
```suggestion
* [CODE_OF_CONDUCT.md](https://github.com/kyma-project/template-repository/blob/main/CODE_OF_CONDUCT.md) - This document is a ready-to-use template which provides a link to the general `CODE_OF_CONDUCT.md` document from the `community` repository.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 8]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
 
-* [CODEOWNERS](https://github.com/kyma-project/community/tree/main/templates/repository-template/CODEOWNERS) - In this document, specify the owners of particular parts of your repository. The owners are automatically added as reviewers when you open a pull request that modifies the code and content they own. If you additionally [modify the settings](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/) of the `main` branch and select the **Require review from Code Owners** option, their approvals become obligatory to merge the pull request. Configure the `CODEOWNERS` document and adjust your `main` branch. The `CODEOWNERS` document contains instructions on how to do both properly.
+* [OWNERS](https://github.com/kyma-project/template-repository/OWNERS) - In this document, specify the owners of particular parts of your repository. The owners are automatically added as reviewers when you open a pull request that modifies the code and content they own.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:20]:
```suggestion
* [OWNERS](https://github.com/kyma-project/template-repository/blob/main/OWNERS) - In this document, specify the owners of particular parts of your repository. The owners are automatically added as reviewers when you open a pull request that modifies the code and content they own.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 9]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
 
-* [CONTRIBUTING.md](https://github.com/kyma-project/community/blob/main/templates/repository-template/CONTRIBUTING.md) - This template makes a reference to the [contributing rules](../../contributing/02-contributing.md) that contain the general guidance from the `community` repository and describes the rules for contributing to all Kyma repositories. If there is any additional, project-specific information that you want to add to your project's `CONTRIBUTING.md` document, add them under the same sections as in the general contributing guide.
+* [OWNERS_ALIASES](https://github.com/kyma-project/template-repository/OWNERS_ALIASES) - This file contains the aliases that the GitHub usernames are grouped. Use it when you want to use group names for assuming the ownership across the repository.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:22]:
```suggestion
* [OWNERS_ALIASES](https://github.com/kyma-project/template-repository/blob/main/OWNERS_ALIASES) - This file contains the aliases that group the GitHub usernames. Use it when you want to use group names for granting ownership across the repository.
```

File: docs/guidelines/repository-guidelines/02-repository-template.md [hunk 10]
```diff
@@ -3,26 +3,28 @@ title: Repository template
...
 
-* [LICENSE](https://github.com/kyma-project/community/tree/main/templates/repository-template/LICENSE) - It is an obligatory element of every open-source repository. Copy the template into your repository.
+* [CONTRIBUTING.md](https://github.com/kyma-project/template-repository/CONTRIBUTING.md) - This template makes a reference to the [contributing rules](../../contributing/02-contributing.md) that contain the general guidance from the `community` repository and describes the rules for contributing to all Kyma repositories. If there is any additional, project-specific information that you want to add to your project's `CONTRIBUTING.md` document, add them under the same sections as in the general contributing guide.
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/02-repository-template.md:24]:
```suggestion
* [CONTRIBUTING.md](https://github.com/kyma-project/template-repository/blob/main/CONTRIBUTING.md) - This template makes a reference to the [contributing rules](../../contributing/02-contributing.md) that contain the general guidance from the `community` repository and describes the rules for contributing to all Kyma repositories. If there is any additional, project-specific information that you want to add to your project's `CONTRIBUTING.md` document, add them under the same sections as in the general contributing guide.
```

File: docs/guidelines/repository-guidelines/01-new-repository-settings.md [hunk 11]
```diff
@@ -9,11 +9,13 @@ All repositories in `kyma-project` and `kyma-incubator` organizations should be
...
+When you create a new repository use the template `kyma-project/template-repository`. This template contains all necessary configuration files for OWNERS, Kyma stale bot, issue and pull request templates, license and code of conduct.
 
-After copying the folder's contents, modify these according to the instructions they contain:
-- [`CODEOWNERS`](https://github.com/kyma-project/community/tree/main/templates/repository-template/CODEOWNERS) to define who is responsible for the review and approval of specific repository parts
-- [`README.md`](https://github.com/kyma-project/community/tree/main/templates/repository-template/README.md) to describe the repository and explain how to use and develop it
+> **NOTE:** Do not check *Include all branches* option!
+
+![Template](./assets/template.png)
```
[the remark this answers is not in the document]
Comment (Ressetkk) [reply docs/guidelines/repository-guidelines/01-new-repository-settings.md:16]: image

File: docs/guidelines/repository-guidelines/01-new-repository-settings.md [hunk 12]
```diff
@@ -91,9 +51,35 @@ If you define any governance-related [Prow job](https://github.com/kyma-project/
 
 ## Create labels
 
-[Define labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories such as [`kyma`](https://github.com/kyma-project/kyma/labels).
+Labels are managed by Prow in the [`labels.yaml`](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml) configuration file.
+To add a new label, create a pull request to the [`test-infra`](https://github.com/kyma-project/test-infra) repository with a new label definition.
+
+This file is split into two sections:
```
Comment (klaudiagrz) [inline docs/guidelines/repository-guidelines/01-new-repository-settings.md:57]: Remove this paragraph, as discussed.

**Review by klaudiagrz — APPROVED** [review]
YAY! 😃

**Pull request merged**